src/components/SeriesSearch/SearchForm.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 
-export const SearchForm = ({ handleChange, handleSubmit, value }) => {
+export const SearchForm = ({ onSubmit }) => {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const inputValue = event.target.querySelector('input[type="text"]').value;
```

Comment:
not very react-like. use refs instead

```
import React, { useRef } from 'react';

export const SearchForm = ({ onSubmit }) => {
  const textRef = useRef();
  const handleSubmit = (event) => {
    event.preventDefault();
    const inputValue = textRef.current.value;
    onSubmit(inputValue);
    event.target.reset();
  };

  return (
    <form className="searchForm" onSubmit={handleSubmit}>
      <input placeholder="Search Series Here" type="text" ref={textRef} />
      <input type="submit" />
    </form>
  );
};
```

reference:
https://medium.com/trabe/react-useref-hook-b6c9d39e2022